src/main/java/dk/kb/util/other/StringListUtils.java

```diff
@@ -0,0 +1,252 @@
+            String s = firstIterator.next();
+            for (String t : list) {
+                if (s.equals(t)) {
+                    continue;
+                }
+                if (t.contains(s)) {
+                    firstIterator.remove();
+                    break;
+                }
+            }
+        }
+        
+        return list;
+    }
+    
+    @SafeVarargs
+    public static <T> Set<T> setOf(T... objects) {
+        return new HashSet<T>(Arrays.asList(objects));
+    }
+    
+    //
+    //public static Set<String> removeSubstrings(Set<String> list) {
+    //
+    //    Set<String> result = new HashSet<>();
+    //    for (String s : list) {
+    //        boolean substring = false;
+    //        for (String t : list) {
+    //            if (s.equals(t)){
+    //                continue;
+    //            }
+    //            if (t.contains(s)){
+    //                substring = true;
+    //                break;
+    //            }
+    //        }
+    //        if (!substring){
+    //            result.add(s);
+    //        }
+    //    }
+    //    return result;
+    //}
+    private static List<String> removeEmpties(String... list) {
+        return Arrays.stream(list)
+                     .filter(Objects::nonNull)
+                     .filter(number -> !number.trim().isEmpty())
+                     .distinct()
+                     .collect(Collectors.toList());
+    }
+    
+    public static String orDefault(String value, String default_value) {
+        if (value == null || value.isBlank()) {
+            return default_value;
+        } else {
+            return value;
+        }
+    }
+    
+    
+    public static <T> T orDefault(T value, T default_value) {
+        return Optional.ofNullable(value).orElse(default_value);
+    }
+    
+    /**
+     * If the string contains /, return the part BEFORE the last /. Otherwise return the entire string
+     *
+     * @param string the string
+     * @return see above
+     */
+    public static String firstPart(String string) {
+        string = notNull(string);
+        
+        if (string.contains("/")) {
+            string = string.substring(0, string.indexOf("/"));
+        }
+        return string;
+    }
+    
+    /**
+     * Return the elements as a MODIFIABLE list
+     * @param strings the elements
+     * @param <T> the type of elements
+     * @return the elements as a ArrayList
+     */
+    public static <T> List<T> toList(T... strings) {
+        if (strings == null) {
+            return new ArrayList<>();
+        } else {
+            return new ArrayList<>(Arrays.asList(strings));
+        }
+    }
+    
+    /**
+     * Trims all the strings in the list and filters out any blank strings and nulls.
+     * @param listOfStrings the strings to be cleaned. If null, returns empty list
+     * @return the resulting list.
+     */
+    public static List<String> cleanList(List<String> listOfStrings) {
```

Comment:
Why does this co-exist with `removeEmpties`? What is the difference?

Reply:
Removed. My great mind have thought out the same function twice at two different times....